ONSdigital/census-rh-ui
# Change for user using welsh url with English region code

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

Resolves CR547 - Language override on EQ Launch

# Motivation and Context
Requested to override EQ payload language_code value when user has an England region code UAC, but uses the welsh url to prevent mixed content in EQ. User also shown a message in welsh warning them that they can only complete in English

# What has changed
Template, translations and eq payload updates

# How to test?
Use welsh url to input UAC, but with an English UAC - see new message and en as language_code in eq_payload
No other combination should show message, and should honour the page/set language code

# Links

# Screenshots (if appropriate):
